internal/pkg/crawl/crawl.go

```diff
@@ -125,6 +126,20 @@ func (c *Crawl) Start() (err error) {
 		go c.startAPI()
 	}
 
+	// Verify that dependencies exist on the system
+	if !c.NoYTDLP {
```

Comment:
instead of adding (another) code block to `crawl.Start()` maybe make a function in `dependencies` package that check if dependencies are satisfied when enabled for the crawl